createNativeWrapper.js

```diff
@@ -42,9 +43,20 @@ export default function createNativeWrapper(Component, config = {}) {
       },
       { ...config } // watch out not to modify config
     );
+    const _ref = useRef();
```

Comment:
I don't quite understand the logic here, could you please explain why current forwarding isn't working and how this exactly works?

I'd also appreciate some example I can test this change on.

Reply:
[filterConfig in createHandler](https://github.com/software-mansion/react-native-gesture-handler/blob/4ce48595cf21ffde161e393c9aa9ee40f484a6e3/createHandler.js#L104)

Reply:
### Context
[`transformIntoHandlerTags`](https://github.com/software-mansion/react-native-gesture-handler/blob/master/createHandler.js#L91) creates an array of valid refs based on the value of the property `_handlerTag` of each ref passed to `waitFor` or `simultaneousHandlers` prop. The filtered array is then sent to native for configuration.
### `NativeViewGestureHandler` 
In the case of `NativeViewGestureHandler` the `_handlerTag` is set on the `NativeViewGestureHandler` ref properly.
**HOWEVER**, this value isn't passed to the ref of it's child. The child's ref is the one being passed back as the ref handler of the component. This makes sense.
**However**, in order to configure interactions on a different handler one would need to pass the `NativeViewGestureHandler` ref as part of `waitFor` or `simultaneousHandlers` prop. 
This is **IMPOSSIBLE** because the `NativeViewGestureHandler` ref isn't exposed.
The child's ref is passed instead and is filtered out by `transformIntoHandlerTags` due to the lack of `_handlerTag` property, making it completely useless.
### The End
So I claim that for a long while (I ran into this bug more than a year back and was confused about it) there wasn't a way to configure interactions on handlers with `NativeViewGestureHandler` as  `waitFor` or `simultaneousHandlers`.